pages/overview/visualizations/issue_assignment.py

```diff
@@ -0,0 +1,273 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.issue_assignee_query import issue_assignee_query as iaq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+
+PAGE = "overview"
+VIZ_ID = "issue_assignment"
+
+gc_issue_assignment = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Issue Assignment Status Counts",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody(
+                            """
+                            Visualizes the number of assigned and unassigned issues in each \n
```

Comment:
Reading the analysis, my takeaway is that this info section isn't 100% correct. We're not counting, at the end of the time window, how many issues are in the 'assigned' state, vs. the number in the 'unassigned' state. 

We're counting, over the duration of the time window, how many state changes occurred into both targets, coming from the shape of the DataFrame that lists all 'assignment' and 'unassignment' events that are within the date-range specified.

To be ultra-clear, for the same issue, multiple 'assign' and 'unassign' events can happen within the time range, at least from my understanding of the data. Since we're doing a left-outer join on the issues table, each issue will be represented multiple times, once for every matching event, or once with a null value for events if no events exist.

Therefore, the info section should reflect that we're really counting events:

"Visualizes the number of times issues were assigned and unassigned to contributors for each time-bin."